Dalamud/Plugin/DalamudPluginInterface.cs

```diff
@@ -87,7 +87,23 @@ public class DalamudPluginInterface : IDisposable {
 #else
         public bool IsDebugging => this.dalamud.DalamudUi.IsDevMenu;
 #endif
+        
+        /// <summary>
+        /// Event that gets fired when loc is changed
+        /// </summary>
+        public event LanguageChangedDelegate OnLanguageChanged;
+        
+        /// <summary>
+        /// Delegate for localization change with two-letter iso lang code
+        /// </summary>
+        /// <param name="langCode"></param>
+        public delegate void LanguageChangedDelegate(string langCode);
 
+        /// <summary>
+        /// Current ui language in two-letter iso format
+        /// </summary>
+        public string UiLanguage;
```

Comment:
Can you make this a "private set" property?

Reply:
Fixed,

Reply:
Looks good, thank you kindly